spec/generators/pageflow/assets/assets_generator_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'spec_helper'
+require 'support/generators'
+require 'fileutils'
+
+# Generators are not automatically loaded by Rails
+require 'generators/pageflow/assets/assets_generator'
+
+module Pageflow
+  module Generators
+    describe AssetsGenerator, type: :generator do
+      setup_default_destination
+
+      before do
+        FileUtils.mkdir_p "#{destination_root}/app/assets/javascripts"
+        FileUtils.mkdir_p "#{destination_root}/app/assets/stylesheets"
+        FileUtils.mkdir_p "#{destination_root}/config"
+
+        FileUtils.touch "#{destination_root}/app/assets/javascripts/active_admin.js.coffee"
```

Comment:
Sounds fine. This could only become a problem, when a generator tries to insert a line after an existing one. But I don't think we are doing this at the moment.